core/test/financial-templates/PricelessPositionManager.js

```diff
@@ -416,9 +421,12 @@ contract("PricelessPositionManager", function(accounts) {
     // Can't transfer if the target already has a pricelessPositionManager.
     await pricelessPositionManager.create({ rawValue: toWei("150") }, { rawValue: toWei("100") }, { from: sponsor });
     assert(await didContractThrow(pricelessPositionManager.transfer(other, { from: sponsor })));
+
+    // Contract state should not have changed.
+    assert.equal(await pricelessPositionManager.contractState(), PositionStatesEnum.OPEN);
   });
 
-  it("Frozen when pre expiry and undercollateralized", async function() {
```

Comment:
I think this "pre-expiry" was a typo, and I wasn't sure why "undercollateralized" is important here? Whoever wrote this test, can you clarify?

Reply:
@ptare @chrismaree any ideas why we are testing for "undercollateralized" positions here?

Reply:
Might have been just copy pasted or maybe the test changed and we didn't update the name.